generic/tests/cfg/iometer_windows.cfg

```diff
@@ -53,16 +53,16 @@
                 - msi_off:
                     cpu_family = "0xe"
         - pm_test:
-            run_timeout = 1000
+            run_timeout = 1800
```

Comment:
no need change run_timeout if not considering win7 guest.

Reply:
ok, will update it